### PR TITLE
13.0 merge from 12.0

### PR DESCRIPTION
**Pull request by fedoranvar**

### CODE REVIEW

File: openapi/doc/changelog.rst [hunk 1]
```diff
@@ -1,3 +1,13 @@
+`1.1.2`
+-------
+
+- **Improvement:** Security rules for reading and configuration
```
Comment (Ramil-Mukhametzyanov) [inline openapi/doc/changelog.rst:4]: :+1: checked

File: openapi/doc/changelog.rst [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+
+`1.1.1`
+-------
+
+- **Fix:** Translation model's float-field into JSON's integer
```
Comment (Ramil-Mukhametzyanov) [inline openapi/doc/changelog.rst:9]: :+1: checked

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

File: .travis.yml [hunk 3]
```diff
@@ -26,7 +26,7 @@ env:
 
 install:
   - pip install anybox.testing.openerp
-  - git clone https://github.com/it-projects-llc/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - git clone https://github.com/fedoranvar/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
```
Comment (yelizariev) [inline .travis.yml]: :-1:

File: openapi/static/src/xml/dashboard.xml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+                    </div>
+                    <t t-if="!ns_last">
+                        <hr/>
+                    </t>
+                </t>
+
+            </t>
+        </div>
+    </t>
+</templates>
```
Comment (yelizariev) [inline openapi/static/src/xml/dashboard.xml]: :-1: file is not needed in odoo 13

File: openapi/tests/test_dashboard.py [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+            '/web',
+
+            "odoo.__DEBUG__.services['web_tour.tour']"
+            ".run('openapi_dashboard')",
+
+            "odoo.__DEBUG__.services['web_tour.tour']"
+            ".tours.openapi_dashboard.ready",
+
+            login='demo'
+        )
```
Comment (yelizariev) [inline openapi/tests/test_dashboard.py]: :-1: the same

**Review by yelizariev — CHANGES_REQUESTED** [review]

**Review by yelizariev — APPROVED** [review]

**Comment by yelizariev** [comment]
I approve to merge it now

**Pull request merged**